test/parallel/test-process-emit-experimentalwarning.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const exec = require('child_process').exec;
+
+if (process.argv[2] === 'child') {
```

Comment:
Even better:

```js
common.expectWarning('ExperimentalWarning', message);
process.emitExperimentalWarning('new_feature');
process.emitExperimentalWarning('new_feature');
```

(call it twice to make sure it is not emitted more than once)